public/js/main.js

```diff
@@ -163,7 +166,9 @@ const initialize = async () =>
  */
 const main = async () => {
   try {
-    await initialize();
+    const [backend] = await initialize();
```

Comment:
It feels a bit odd that initialize would only return backend (what about all the other things that get inited?) How would you feel about passing state to initialize (and subsequently initializemodels) and having it set state.backend if it successfully initializes a backend?

Reply:
Yeah, I get you.
Well, I think it's indeed a good idea (although passing `state` to `initialize` doesn't really make sense since it's a global variable).